sparql-ns.ttl

```diff
@@ -0,0 +1,403 @@
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-contains> ;
+    .
+
+sparql:strbefore rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-strbefore> ;
+    .
+
+sparql:strafter rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-strafter> ;
+    .
+
+sparql:concat rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-concat> ;
+    .
+
+sparql:langMatches rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-langMatches> ;
+    .
+
+sparql:regex rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-regex> ;
+    .
+
+sparql:replace rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-replace> ;
+    .
+
+sparql:encode rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-encode> ;
+    .
+
+# Section: Functions on numbers
+
+sparql:abs rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-abs> ;
+    .
+
+sparql:round rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-round> ;
+    .
+
+sparql:ceil rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-ceil> ;
+    .
+
+sparql:floor rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-floor> ;
+    .
+
+# Section: Functions on datetimes
+
+sparql:now rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-now> ;
+    .
+
+sparql:year rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-year> ;
+    .
+
+sparql:month rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-month> ;
+    .
+
+sparql:day rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-day> ;
+    .
+
+sparql:hours rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-hours> ;
+    .
+
+sparql:minutes rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-minutes> ;
+    .
+
+sparql:seconds rdf:type sparql:Function ;
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/#func-seconds> ;
+    .
+
+sparql:timezone rdf:type sparql:Function ;
+    dc:description "" ;
```

Comment:
This function returns the timezone component as a `dayTimeDuration` for an `xsd:dateTime` value with a time zone specified, or an empty value if none.

Reply:
```suggestion
    dc:description "This function returns the timezone component as a dayTimeDuration for an xsd:dateTime value with a specified time zone, or an empty value if none." ;
```